src/core-services/account/resolvers/Group/createGroup.js

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
If this is the mutation resolver for `createGroup` then it's not in the correct place. It should be in the `resolvers/Mutation` and exported from `resolvers/Mutation/index.js`.